Tests/PureeTests/Output/BufferedOutputTests.swift

```diff
@@ -169,7 +169,7 @@ class BufferedOutputTests: XCTestCase {
         }
         output.resume()
 
-        let expectedWriteCount = Int(ceil(Double(testIndices.count) / Double(output.configuration.logEntryCountLimit)))
+        let expectedWriteCount = 67
```

Comment:
This number comes from `200 / 3`? lol.

How about modify `retryLimit`?
https://github.com/cookpad/Puree-Swift/pull/11/commits/e6e93d78111288dc9ca3265bcceb6db070bebec8#diff-bd451c976c20bf0bfe90238f1b438e5fR147